Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+#!groovy
+@Library('jenkins-pipeline-library') _
+
+def label = "filler-${UUID.randomUUID().toString()}"
+
+podTemplate(label: label, yaml: readTrusted('JenkinsPods.yaml')) {
+  node(label) {
+
+    ci {
+      gitCheckout()
+
+      stage('download Go deps') {
+        container('golang'){
+          sh 'apk add --no-cache git gcc libc-dev'
```

Comment:
why do you install gcc and libc since CGO is disabled https://github.com/AirHelp/filler/blob/master/Dockerfile-test#L3??